compute_funcx/sdk/PyPI.md

```diff
@@ -0,0 +1,11 @@
+# Deprecation Warning
+
+This package is deprecated, and currently just wraps the [Globus Compute SDK](https://pypi.org/project/globus-compute-sdk/) package with FuncX names. Users are recommended to upgrade to Globus Compute instead of using this package.
```

Comment:
Link to the funcx_upgrade page probably useful.  (https://globus-compute.readthedocs.io/en/latest/funcx_upgrade.html)